tox.ini

```diff
@@ -8,8 +8,8 @@ requires = pip
 # We need this to be allowed to use the timeout command
 allowlist_externals=timeout
 deps=
+    .
     -rrequirements.dev.txt
-    -rrequirements.txt
```

Comment:
?

Reply:
I thought it was enough to install inmanta-core using `.` (setup.py)